pages/linux/btrfs-balance.md

```diff
@@ -0,0 +1,32 @@
+# btrfs balance
+
+> Balance block groups on a btrfs filesystem.
+> More information: <https://btrfs.wiki.kernel.org/index.php/Manpage/btrfs-balance>.
+
+- Show the status of a running or paused balance:
+
+`sudo btrfs balance status {{path/to/btrfs_filesystem}}`
+
+- Balance all block groups (slow; rewrites all blocks in filesystem):
+
+`sudo btrfs balance start {{path/to/btrfs_filesystem}}`
+
+- Balance data block groups less than 15% utilized (and run in background):
+
+`sudo btrfs balance start --bg -dusage=15 {{path/to/btrfs_filesystem}}`
+
+- Balance (max 10) metadata chunks with < 20% utilization and at least 1 chunk on device `devid` (`btrfs filesystem show`):
```

Comment:
Perhaps this will read better?
```suggestion
- Balance a max of 10 metadata chunks with less than 20% utilization and at least 1 chunk on a given device `devid` (see `btrfs filesystem show`):
```